packages/picker/test/index.ts

```diff
@@ -207,44 +199,17 @@ export function runPickerTests(): void {
             };
 
             let name = 'Where do you live?';
-            const isWebKitLinux = isWebKit() && detectOS() === 'Linux';
 
-            // WebKit (in CI) doesn't currently associate the `name` via the accessibility tree.
```

Comment:
Gotta love these comments telling me exactly what is going on here and why I can now delete them. 🥰